src/Swarm/Game/Scenario.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE DeriveAnyClass #-}
 {-# LANGUAGE DeriveFunctor #-}
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
It's odd hlint does not complain about unused extensions, but I think that's fine because we'll probably need them for the scenarios.